views/js/jquery.rating.plugin.js

```diff
@@ -100,7 +100,7 @@ jQuery.fn.rating = function(generalOptions) {
         fullStars.append(newStar);
       }
 
-      fullStars.hover(function(){}, displayInteractiveGrade);
+      fullStars.mouseenter(function(){}).mouseleave(displayInteractiveGrade);
```

Comment:
```suggestion
      fullStars.on('mouseleave', displayInteractiveGrade);
```
Hi @leemyongpakvn, I think you can simplify more here.